api/XDS_PROTOCOL.md

```diff
@@ -345,6 +345,14 @@ resources to avoid resending them over the network.
 
 ![Incremental reconnect example](diagrams/incremental-reconnect.svg)
 
+#### Resource names
+Resources are identified by a resource name or an alias. Aliases of a resource, if present, can be identified by the alias field in the resource of a DeltaDiscoveryResponse. The resource name
+will be returned in the name field in the resource of a DeltaDiscoveryResponse. Envoy can send either an alias or the resource name in the resource_names_subscribe field in order to subscribe to a resource.
+
+#### Unsubscribing from Resources
+Envoy will keep track of a per resource reference count internally. This count will keep track of the total number of aliases/resource names that are currently subscribed to. When the reference count reaches zero,
+Envoy will send a DeltaDiscoveryRequest containing the resource name of the resource to unsubscribe from in the resource_names_unsubscribe field.
```

Comment:
Ditto here. Also, can you do nice line wrap for all this MD text? Maybe at 100 chars?

Reply:
Thanks updated.